cmd/monaco/integrationtest/v2/delete_integration_test.go

```diff
@@ -324,6 +325,9 @@ configs:
 	// DEPLOY Config
 	err = monaco.Run(t, fs, fmt.Sprintf("monaco deploy %s --verbose", deployManifestPath))
 	require.NoError(t, err)
+
+	time.Sleep(10 * time.Second)
+
 	integrationtest.AssertAllConfigsAvailability(t, fs, deployManifestPath, []string{}, "", true)
```

Comment:
Why do we call `integrationtest.AssertAllConfigsAvailability(...)` twice, once here and once on 346?